cmd/ops_agent_windows/main_windows.go

```diff
@@ -36,8 +36,29 @@ var (
 	healthChecks      = flag.Bool("healthchecks", false, "run health checks and exit")
 )
 
+type WindowsDefaultLogger struct {
+	l *log.Logger
+}
+
+func (wdl WindowsDefaultLogger) Printf(format string, v ...any) {
+	wdl.l.Printf(format, v...)
+}
+
+func (wdl WindowsDefaultLogger) Infof(format string, v ...any) {
+	wdl.l.Printf(format, v...)
+}
+
+func (wdl WindowsDefaultLogger) Errorf(format string, v ...any) {
+	wdl.l.Printf(format, v...)
+}
+
+func (wdl WindowsDefaultLogger) Println(v ...any) {
+	wdl.l.Println(v...)
+}
+
 func main() {
-	infoLog := log.New(os.Stdout, log.Prefix(), log.Flags())
+
+	infoLog := WindowsDefaultLogger{log.New(os.Stdout, log.Prefix(), log.Flags())}
```

Comment:
Nit : Maybe the `log.New(os.Stdout, log.Prefix(), log.Flags())` can be moved inside the `WindowsDefaultLogger` definition.